README.md

```diff
@@ -1 +1,44 @@
-# dnanexus_RPKM
+# dnanexus_RPKM v1.0
+
+## What does this app do?
+This app uses conifer to perform RPKM analysis.
+
+## What are typical use cases for this app?
+This app has been designed to be run in an automated fashion following the end of a sequencing run.
+
+Using the inputs described below BAM files for each panel code are downloaded.
+A minimum of three samples of the same panel must be present in the project. 
+
+
+## What data are required for this app to run?
+* A BED file for the CNV regions 
+  * This bedfile must NOT have a header
+  *  Each entry must have  'chr' removed eg:
+      `11    108093508    108093963    ATM_Ex01_PM`
+  *  An app (RPKM_bedfile) has been created which can be used to convert the data.bed bed file produced by mokabed into the desired format (RPKM.bed)
+* A project in which to find BAM files
+  * BAM files must be found in /output/
+* The variant calling bed file Pan number
+  * The RPKM bed file will have a different panel number (for +/-50bp) therefore the panel number which corresponds to the +/-10bp panel is required to download the relevant BAM files.  
```

Comment:
In moka there is entry in the gene panel table (which creates the Pannumber) for +/-10 and another for +/-50 which also details the control sites for CNV analysis.